keps/sig-node/4381-dra-structured-parameters/README.md

```diff
@@ -419,51 +398,46 @@ As a cluster administrator, I want to make GPUs from vendor ACME available to us
 of that cluster. I prepare the nodes and deploy the vendor's components with
 `kubectl create`.
 
-I create a ResourceClass for the hardware with parameters that only I as the
+I create a DeviceClass for the hardware with parameters that only I as the
```

Comment:
This use case is the opposite: it *grants* users access to settings that they wouldn't be able to use otherwise.

What you have in mind is the use case "I as administrator want to ensure that all devices are configured in a certain way." Do we really have that use case?

Isn't it better solved when deploying the driver? The driver has to support configuration options for whatever the admin wants to control already for this to work, so this could also be part of the driver-specific deployment.

Does it have to be through the API? 

Both use cases may need further justifications. Perhaps we should remove config from class entirely in 1.31 and defer?

Reply:
> >  Isn't it better solved when deploying the driver?
>
> Given my own context of a cloud provider, I doubt very much that cluster admins will be involved in the deployment of drivers. They will come with the VMs and be managed by the cluster provider.

Ack. However, we still don't know what that admin-controlled configuration would be, so perhaps it is too early to design an API for this.

> I am also still very concerned about quota-ish admin controls and I see class playing a big role in that.

Let's discuss quota in https://github.com/kubernetes-sigs/wg-device-management/issues/24